src/js/components/tasks/table.tsx

```diff
@@ -239,7 +263,7 @@ const AssigneeTableCell = ({
         />
         <AssignTaskRoleModal
           projectId={projectId}
-          epicUsers={epicUsers}
+          epicUsers={epicUsers || []}
```

Comment:
Now that we have access to the list of GitHub user ids for each epic (via `item.epic.github_users`), we can replace the `|| []` here and instead populate a real list of `epicUsers` if its missing.

To populate that list (getting a list of full user objects instead of a list of `id` strings), see the logic in `useFetchEpicIfMissing.tsx` (https://github.com/oddbird/Metecho/blob/view-all-tasks/src/js/components/utils/useFetchEpicIfMissing.tsx#L24-L26). It'll involve filtering `githubUsers` (which is the list of all project users, as full user objects) down to only the ones who have ids that match `item.epic.github_users`.